### DISCUSSION TITLE
Fatal error while trying to run verdaccio container using docker-compose.yml

### DISCUSSION THREAD

**Discussion by ggehlot1980** [verdaccio/verdaccio#2348] [Q&A] [answered]
following is my docker-compose.yml

version: '3.1'
services:
  verdaccio:   
    image: verdaccio/verdaccio
    container_name: "verdaccio"
    networks:
      - node-network
    environment:
      - VERDACCIO_PORT=4873
    ports:
      - "4873:4873"
    volumes:
      - ./storage:/verdaccio/storage
      - ./config:/verdaccio/conf
      - ./plugins:/verdaccio/plugins
      - ./logs:/verdaccio/logs
networks:
  node-network:
    driver: bridge
______________________________________________________________________________________
On running
"sudo docker-compose.yml up"
the container gets created but stops after logging the below error:
fatal--- cannot open config file /verdaccio/conf/config.yaml: Error: CONFIG: it does not look like a valid config file

**Comment by juanpicado** [comment, marked as answer]
Check volumen permissions in your folder, check verdaccio docs (docker page) for it.
